lib/DataSift/ApiClient.php

```diff
@@ -67,7 +67,12 @@ static public function call(
         $ssl = $user->useSSL();
 
         // Build the full endpoint URL
-        $url = 'http'.($ssl ? 's' : '').'://'.$user->getApiUrl(). $user->getApiVersion() . '/'. $endPoint;
+        if ($method == 'ingest') {
```

Comment:
Yeah that could work, I'll take a look tomorrow unless you wanted to take a look.